szperacz/run.py

```diff
@@ -17,6 +20,13 @@ def log_to_console(files):
         print(f'{i}')
 
 
+def print_to_json_file(files):
+    file = open('output.txt', 'w')
```

Comment:
1. Please rename the function to something more like: `log_data_to_json`, `log_to_file`, etc.
2. Please change the file name to: `data.json`. Name is not that important, but keeping the extension `.json` strict as the file format is critical. `.json` files are also text files for Linux and Windows.
3. Do no use `open`/`close` function pair. It is much better to use `with open(...` structure. See last example here: https://stackabuse.com/writing-to-a-file-with-pythons-print-function/